content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -350,6 +350,10 @@ Starting with kubernetes v1.11, kubelet and kube-proxy can run as Windows servic
 
 This means that you can now register them as Windows services via `sc` command. More details about how to create Windows services with `sc` can be found [here](https://support.microsoft.com/en-us/help/251192/how-to-create-a-windows-service-by-using-sc-exe).
 
+### GMSA support
+
+Starting with Kubernetes v1.14, Windows container workloads can be configured with a GMSA credential spec JSON. Operators can configure a set of GMSA credential specs that can be used in Windows worker nodes across the Kubernetes cluster 
```

Comment:
I'd use a [feature state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) here.

Reply:
Should we expand GMSA to Group Managed Service Account?